### ISSUE TITLE
Blank Screen in Chrome Extension v.7.2.1

### ISSUE THREAD

**Issue by gregzaitsev** [MetaMask/metamask-extension#7195]
**Describe the bug**
Metamask shows blank screen after I login.

**To Reproduce**

1. Open Metamask first time. Go through creation of new account
2. Close Metamask for a long time (or turn off and then back on) to cause logout
3. Metamask asks for password. Enter it.
4. Metamask asks for password again, but on a differently looking screen. Enter it.
5. Metamask shows blank screen forever (waited 10+ minutes)

**Expected behavior**

**Screenshots**

**Browser details (please complete the following information):**
 - OS: Ubuntu 16.04
 - Chrome Browser: 77.0.3865.90
 - MetaMask Version 7.2.1

**Additional context**
Metamask settings: Site access - on all sites, collect errors enabled. Other options disabled.

**Comment by gregzaitsev** [comment]
Step 3:
![image](https://user-images.githubusercontent.com/2152909/65252792-3d842a00-db02-11e9-8cb2-90403b971b6d.png)

Step 4:
![image](https://user-images.githubusercontent.com/2152909/65252839-5260bd80-db02-11e9-8858-ed7e578422fa.png)

Step 5: 
![image](https://user-images.githubusercontent.com/2152909/65252881-62789d00-db02-11e9-9f6a-f9b787623b8c.png)

**Comment by gregzaitsev** [comment]
There is no error output in background page console. All I see is this:

![image](https://user-images.githubusercontent.com/2152909/65253007-92c03b80-db02-11e9-83d5-1c0b4febfb60.png)

**Comment by gregzaitsev** [comment]
I also see this identical behavior on a different computer with different version of Google Chrome (75), but same version of Metamask and same Ubuntu 16.04.

**Comment by danfinlay** [comment]
That second password screen is the same as the first, but the CSS styles have broken, because some kind of crash has already occurred.

Did you check the logs in the UI console?

It would be good to get state logs from this event, they might help. If you want maximal privacy, you can send them to support@metamask.io

To help us diagnose your issue, you can download your "state logs" by [following this guide](https://metamask.zendesk.com/hc/en-us/articles/360015290092-How-To-Get-Logs-And-Help-MetaMask-Support-and-Diagnose-Your-Issue), and then sending them to support@metamask.io .

**Comment by dpazdan** [comment]
I was unable to reproduce this on:

OS: Ubuntu 18.04.2 (newer)
Chrome Browser: 77.0.3865.90 (same)
MetaMask Version 7.2.1 (same)

**Comment by Gudahtt** [comment]
I wasn't able to reproduce this exactly, but I did reproduce something vaguely similar.

I found an error in Sentry that matches this description somewhat, and it resulted from this error in the UI:

```
BigNumber Error: times() not a number: null
  at raise(node_modules/bignumber.js/bignumber.js:1190:1)
  at parseNumeric(node_modules/bignumber.js/bignumber.js:1178:1)
  at new K(node_modules/bignumber.js/bignumber.js:193:1)
  at apply(node_modules/bignumber.js/bignumber.js:2092:1)
  at apply(node_modules/ramda/src/invoker.js:38:1)
  at apply(node_modules/ramda/src/internal/_curryN.js:37:1)
  at f(node_modules/ramda/src/internal/_arity.js:5:1)
  at toFunctorFn(node_modules/ramda/src/over.js:37:1)
  at lens(node_modules/ramda/src/lens.js:35:1)
  at fn(node_modules/ramda/src/over.js:37:1)
  at fn(node_modules/ramda/src/internal/_curry3.js:21:1)
  at _b(node_modules/ramda/src/internal/_curry2.js:25:1)
  at apply(node_modules/ramda/src/converge.js:40:1)
  at f1(node_modules/ramda/src/internal/_curry1.js:17:1)
  at fn(node_modules/ramda/src/when.js:33:1)
  at apply(node_modules/ramda/src/internal/_curry3.js:26:1)
  at f1(node_modules/ramda/src/internal/_curry1.js:17:1)
  at apply(node_modules/ramda/src/internal/_pipe.js:3:1)
  at apply(node_modules/ramda/src/internal/_pipe.js:3:1)
  at apply(node_modules/ramda/src/internal/_pipe.js:3:1)
  at apply(node_modules/ramda/src/internal/_pipe.js:3:1)
  at apply(node_modules/ramda/src/internal/_pipe.js:3:1)
  at apply(node_modules/ramda/src/internal/_pipe.js:3:1)
  at converter(node_modules/ramda/src/internal/_arity.js:5:1)
  at conversionUtil(ui/app/helpers/utils/conversion-util.js:127:7)
  at getValueFromWeiHex(ui/app/helpers/utils/confirm-tx.util.js:64:10)
  at mergeProps(ui/app/components/ui/currency-display/currency-display.container.js:31:5)
```

Ultimately it was caused by `conversionRate` being set to null, which I suspect was due to a failed network request being mishandled (hence the lack of an error in the background). This is speculation, but the API we use to get that conversion rate had downtime recently: https://status.cryptocompare.com/status/updates?createdOn=1568910944

We should be able to handle a missing `conversionRate` without crashing of course - we should just not show the converted currency. I've improved the error handling in the controller responsible for the `conversionRate` here: https://github.com/MetaMask/gaba/pull/157

**Comment by gregzaitsev** [comment]
I think I chased the root cause for the issue. When I connect through a VPN, Metamask works, the issue is gone. I am located in Russia and the local Internet provider might have blocked the access to some of the IPs Metamask is trying to access, perhaps even Infura node.

**Comment by valdas** [comment]
The same issue with: 

Linux
Chrome Version 64.0.3282.167 (Official Build) (64-bit)
MetaMask 7.3.1

After login UI does not work. Worked some time ago with older extension version.

No error message or something. User is left by himself with no acces to wallet.

**Comment by tmashuang** [comment]
Fixed in latest version, 7.7.3. If the issue has persisted, please email [support](https://metamask.zendesk.com/hc/en-us/requests/new).
